# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ogle/ogle-unstatic.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ogle-0.9.2/ac3/ac3dec_wrap.c.orig	2005-09-08 15:59:15.000000000 -0700
+++ ogle-0.9.2/ac3/ac3dec_wrap.c	2005-09-08 15:59:19.000000000 -0700
@@ -48,7 +48,7 @@
 static void handle_events(MsgEventQ_t *q, MsgEvent_t *ev);
 
 
-static char *program_name;
+char *program_name;
 
 static FILE *outfile;
 
